EIPS/eip-9999.md

```diff
@@ -0,0 +1,285 @@
+
+Since this attack results in a split view at a cost for the builder (the payload is revealed and may not be included) this EIP opted for simplicity of implementation
+
+
+### Withdrawals
+
+Withdrawals from the beacon chain are complex in nature, they involve removing funds from one layer and crediting them on another, with different trigger mechanisms that can start from either layer. Before applying the consensus layer state transition function to a given beacon state `pre_state` and processing a given signed beacon block `block`, the set of withdrawals that are expected to be deducted from the beacon chain are completely determined by `pre_state`. Previous to this EIP the set of withdrawals that are credited on the execution layer are included in `block`. The block is deemed invalid if these withdrawals do not match. With the separation included in this EIP, these operations of deducting and crediting become asynchronous:
+- When processing the beacon block, the withdrawals are deducted from the beacon chain. 
+- The set of withdrawals just deducted is committed to the beacon state `post_state`.  
+- When processing any execution payload whose parent beacon state is `post_state`, the payload is deemed invalid if it doesn't include precisely the list of withdrawals committed to `post_state`. 
+
+This asynchronous mechanism has some consequences as slots may be *empty* as defined above. In these cases, the consensus layer does not process any more withdrawals until an execution payload has fulfilled the outstanding ones. An alternative design would be to defer all of withdrawal processing to the execution payload validation phase (ie. `process_execution_payload`). This has the advantage of not needing to track the fulfilled withdrawals on the beacon chain. The logic changes when several payloads are missing, in which case balances on the beacon chain change and therefore a withdrawal that would be possible with the former mechanism may be different, or even impossible with the latter.
+
+### Three state transition functions
+
+The current EIP adds an extra state transition function to the block processing in Ethereum. Processing a `SignedBeaconBlock` changes the consensus layer `BeaconState`. A `SignedExecutionPayloadEnvelope` changes both the execution layer state and the consensus layer one. As such, the envelope commits to the consensus layer post-state-transition beacon state root. 
+
+### Compatible designs 
+
+#### Inclusion lists
+
+This EIP is fully compatible with forward inclusion lists as specified in [EIP-7547](./eip-7547.md) or similar.
+
+#### Slot auctions
+
+A simple change to this EIP is to remove the blockhash commitment from the `SignedExecutionPayloadHeader`. This allows the builder to commit any payload to the slot. A preliminary security analysis shows that payload equivocation does not weaken fork choice's FFG. Some advantages of Slot auctions include:
```

Comment:
Should it be 'slot auctions' or 'Slot Auctions'? Not sure why 'slot' is capitalized.

Reply:
because it's a header title.